molecule/default/molecule.yml

```diff
@@ -18,7 +18,6 @@ platforms:
       - 8081-8090:8081-8090/tcp
     networks:
       - name: cartridge-network
-    purge_networks: true
```

Comment:
Is it still necessary after manually installing molecule?

Reply:
What do you mean? 
I think that this option was specified for tests on GitLab CI, now everything works fine without purging networks.

Reply:
`purge_networks` seems useful option which helps isolating a container. But I don't have any strong argument against returning it back.